detox/src/artifacts/screenshot/ScreenshotArtifactPlugin.js

```diff
@@ -17,13 +17,13 @@ class ScreenshotArtifactPlugin extends TwoSnapshotsPerTestPlugin {
     return this.api.preparePathForArtifact(`${name}.png`, testSummary);
   }
 
-  async onBeforeCleanup(e) {
+  async onBeforeCleanup() {
```

Comment:
Hmm... okay?

Reply:
Totally :) there is no argument there at all. Noticed thanks to TypeScript, actually.